src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.css

```diff
@@ -6,3 +6,15 @@
 dashboard-widget-wrapper .action-label {
 	padding: 7px;
 }
+
+dashboard-widget-wrapper .header {
+	font-weight: 600;
+	font-size: 14px;
+	line-height: 20px;
+	color: #323130;
+}
+
+.vs-dark dashboard-widget-wrapper .header,
+.vs-hc dashboard-widget-wrapper .header {
+	color: #FFFFFF;
```

Comment:
i wanted to use color registry for all dashboard related colors. let's review the colors, consolidate them and use color registry.